apps/server/src/shared/repo/task/task.repo.ts

```diff
@@ -193,4 +193,78 @@ export class TaskRepo extends BaseRepo<Task> {
 
 		return [taskEntities, count];
 	}
+
+	// This method is used for syncing the new filerecords collection with the old files collection.
+	// It can be removed after transitioning file-handling to the new files-storage-microservice is completed.
+	async getTasksToSync(batchSize = 50) {
```

Comment:
This is also for the job only. We should not add this to the repo please hold it by the temporay job code.
Same for the test